_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -275,12 +283,53 @@ class HealthCareApp extends React.Component {
           external: this.getExternalData(this.state.applicationData, statePath)
         });
       });
+    }
 
-      if (this.props.location.pathname === '/review-and-submit') {
-        buttons = (<SubmitButton onButtonClick={this.handleSubmit}/>);
-      } else {
-        buttons = (<ContinueButton onButtonClick={this.handleContinue}/>);
-      }
+    // Check which section the user is on and render the correct ProgressButtons.
+    const lastSectionText = (this.getUrl('back')) ? this.getUrl('back').split('/').slice(-1)[0].replace('-', ' ') : '';
+    const nextSectionText = (this.getUrl('next')) ? this.getUrl('next').split('/').slice(-1)[0].replace('-', ' ') : '';
+
+    const backButton = (
+      <ProgressButton
+          onButtonClick={this.handleBack}
+          buttonText={`Back to ${lastSectionText}`}
+          buttonClass={'usa-button-primary-alt'}/>
```

Comment:
What do you think of the styling of `usa-button-primary-alt`? I'm not too crazy about the bright blue... I think when we were initially mocking up this form with @emilyville we talked about making the secondary button with a gray background. Can we try that instead?